config.md

```diff
@@ -69,7 +70,7 @@ For Solaris, the mount entry corresponds to the 'fs' resource in the [zonecfg(1M
   * Windows: a local directory on the filesystem of the container host. UNC paths and mapped drives are not supported.
   * Solaris: corresponds to "special" of the fs resource in [zonecfg(1M)][zonecfg.1m].
 * **`options`** (array of strings, OPTIONAL) Mount options of the filesystem to be used.
-  * Linux: supported options are listed in the [mount(8)][mount.8] man page. Note both [filesystem-independent][mount.8-filesystem-independent] and [filesystem-specific][mount.8-filesystem-specific] options are listed.
+  * Linux: supported options are listed in the [mount(8)][mount.8] man page. Note both [filesystem-independent][mount.8-filesystem-independent] and [filesystem-specific][mount.8-filesystem-specific] options are listed. In the case of bind mounts, this would be the file on the host.
```

Comment:
> Are you saying the statement In the case of bind mounts, this would be the file on the host. is invalid?

Yes.

> It's inconsistent to have it in the go code, so where should it go?

I think the Go comments should be as short and sweet as possible to avoid these inconsistencies.  The more of the spec that gets echoed in the Go comments, the less DRY we are and the more likely we have these inconsistencies.

> Or should it be entirely removed?

Yes, I'd like this PR to remove it from the Go comment but not inject it into the Markdown.